### PR TITLE
Add influent/effluent quality metrics for ASM1

### PR DESCRIPTION
**Pull request by adam-a-a**
## Summary/Motivation:
This PR adds quality metrics to ASM1 which can be used to evaluate WWTP performance and should be useful for optimization cases. Said metrics/properties can now be calculated for any ASM1 stateblock.

## Changes proposed in this PR:
- add TSS
- add BOD5- note- this is the only one indexed by "raw" or "effluent", each of which incorporate a different conversion factor
- add COD
- add Kjeldahl nitrogen
- add total nitrogen
- add associated params as needed and move a couple params from ASM1 rxn to prop model

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/property_models/activated_sludge/asm1_properties.py
Comment (MarcusHolly) [inline watertap/property_models/activated_sludge/asm1_properties.py:425]: Maybe I'm just not understanding, but is there a reason these two are bundled together as opposed to having a separate expression for each?

Comment (adam-a-a) [reply watertap/property_models/activated_sludge/asm1_properties.py:425]: The only difference between the two is that for treated effluent, the whole expression should be multiplied by one value (0.25), while for another stateblock in the flowsheet (e.g., influent, stream bypassing activated sludge process, etc.), the expression is multiplied by another value (0.65). I could've made them separate but chose to do it this way. Note the `BOD5_factor` that I added to the parameter block is also indexed by "raw" or "effluent". (This is the factor that is 0.65 for "raw" and 0.25 for "effluent".)

**Review by MarcusHolly — APPROVED** [review]
LGTM - I'm assuming this PR is done

**Review by agarciadiego — APPROVED** [review]

**Review by luohezhiming — APPROVED** [review]
LGTM. Once it get merged, we can move forward to do some anlysis on these key metrics.

**Pull request merged**
